.travis.yml

```diff
@@ -1,31 +1,40 @@
 language: node_js
-node_js:
-  - "7"
-  - "6"
-  - "4"
-  - "0.12"
-  - "0.10"
-  - "iojs"
 
 compiler: gcc
 sudo: false
 
-os:
-  - linux
-  - osx
-
 env:
   global:
     - SKIP_SASS_BINARY_DOWNLOAD_FOR_CI=true
 
-matrix:
-  fast_finish: true
-  exclude:
-    - node_js: iojs
-    - node_js: "0.10"
+jobs:
+  include:
+    - stage: test
+      node_js: "7"
```

Comment:
Looks like there is a "node" key https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions
We would still have to change for Node 8 since we would need to add it to the matrix. I do like this idea though